### ISSUE TITLE
Expose `has-more` header to caller

### ISSUE THREAD

**Issue by 501st-alpha1** [dpeite/py-glo-board#2]
Some endpoints, such as [Get Cards by Board](https://gloapi.gitkraken.com/v1/docs/#/Cards/get_boards__board_id__cards), return a `has-more` header which indicates whether the results are incomplete and there are more results on the next page.  It doesn't look like this is currently returned to the caller, so it'd be useful to expose that.

Do you have any thoughts on the best way to do this?  If you don't have time to actually implement it yourself, I can probably open a PR after we discuss this.

---

Thanks for making this library by the way!  It has greatly sped up my ability to write scripts for the Glo Boards API.

**Comment by dpeite** [comment]
Hi, It's been a while since I wrote this code and I see that the api has some changes like new endpoints and the `has-more` header.

I'm a bit busy right now, but I'll try to get some time in the next days and look into the new changes.

I suppose that the way to do this is with a method that return if the last call to an endpoint has the header defined. But I'm open to new ideas or suggestions.

**Comment by 501st-alpha1** [comment]
> I suppose that the way to do this is with a method that return if the last call to an endpoint has the header defined. But I'm open to new ideas or suggestions.

Yes that should work.  An alternative would be to return a custom object, with the current result as a member of that object.  So instead of e.g. returning a list of Boards, it'd be something like:
```
ResultObject:
  has_more: True,
  data: <boards...>
```

The downside would be that this wouldn't be backwards-compatible, but this has the benefit of making each function call stateless, and it would probably make it easier to expand to returning other things if needed, such as the HTTP status code.

No rush on the changes either way.  If you don't have time and would like me to attempt a PR, let me know.
